trailblazer/containers.py

```diff
@@ -20,6 +22,9 @@ class Container(containers.DeclarativeContainer):
     google_oauth_base_url: str = os.environ.get("GOOGLE_OAUTH_BASE_URL")
     encryption_key: str = os.environ.get("ENCRYPTION_KEY")
     google_api_base_url: str = os.environ.get("GOOGLE_API_BASE_URL")
+    slurm_jwt_token: str = os.environ.get("SLURM_JWT")
+    slurm_user_name: str = os.environ.get("SLURM_USER_NAME")
+    slurm_base_url: str = os.environ.get("SLURM_BASE_URL")
```

Comment:
What happens if these do not exist? Are the variables just empty strings or do we need to add these in our current configs?

Reply:
Ah, good point. The type hint is wrong, it is a string or none. So no, we do not need to update our configs

Reply:
Fixed type hints in https://github.com/Clinical-Genomics/trailblazer/pull/451/commits/a4d97470c85cca02a7e045d2ab7e38a2670b52b8